platform/android/src/offline/offline_manager.cpp

```diff
@@ -145,5 +173,33 @@ void OfflineManager::CreateOfflineRegionCallback::onCreate(jni::JNIEnv& env,
     callback.Call(env, method, OfflineRegion::New(env, jFileSource, std::move(*region)));
 }
 
+// OfflineManager::MergeOfflineRegionsCallback //
+
+void OfflineManager::MergeOfflineRegionsCallback::onError(jni::JNIEnv& env,
+                                                          const jni::Object<OfflineManager::MergeOfflineRegionsCallback>& callback,
+                                                          std::exception_ptr error) {
+    static auto& javaClass = jni::Class<OfflineManager::MergeOfflineRegionsCallback>::Singleton(env);
+    static auto method = javaClass.GetMethod<void (jni::String)>(env, "onError");
+
+    callback.Call(env, method, jni::Make<jni::String>(env, mbgl::util::toString(error)));
+}
+
+void OfflineManager::MergeOfflineRegionsCallback::onMerge(jni::JNIEnv& env,
+                                                          const jni::Object<FileSource>& jFileSource,
+                                                          const jni::Object<MergeOfflineRegionsCallback>& callback,
+                                                          mbgl::optional<std::vector<mbgl::OfflineRegion>> regions) {
+    static auto& javaClass = jni::Class<OfflineManager::MergeOfflineRegionsCallback>::Singleton(env);
```

Comment:
Does the `regions` parameter need to be an `optional`? If so, there should be a check to see if it actually contains a value.

Reply:
The check is performed beforehand, we can remove the `optional`.